doc/visual-programming/source/widgets/evaluate/calibrationplot.md

```diff
@@ -7,21 +7,41 @@ Shows the match between classifiers' probability predictions and actual class pr
+   - Curves for individual folds: a curve is displayed for each fold from the [Test and Score](testandscore.md) widget.
+2. Choose which classifiers to plot. The diagonal represents optimal behavior when *Calibration curve* is selected; the closer the classifier's curve gets, the more accurate its prediction probabilities are.
+3. Select the metric to calibrate:
+   - *calibration curve*: displays calibration curves for multiple models. It outputs a model with smoothed probabilities (converting them). The options for smoothing functions are [Sigmoid](https://en.wikipedia.org/wiki/Sigmoid_function) or [Isotonic](https://en.wikipedia.org/wiki/Isotonic_regression) function.
+   - *classification accuracy*: displays classification accurracy at changing probability thresholds. Threshold can be set by dragging the vertical line left or right.
+   - *F1*: displays F1 score at changing probability thresholds. Threshold can be set by dragging the vertical line left or right.
+   - *sensitivity and specificity*: displays the relationship between [sensitivity and specificity](https://en.wikipedia.org/wiki/Sensitivity_and_specificity) at changing probability thresholds. Threshold can be set by dragging the vertical line left or right.
+   - *precision and recall*: displays the relationship between [precision and recall](https://en.wikipedia.org/wiki/Precision_and_recall) at changing probability thresholds. Threshold can be set by dragging the vertical line left or right.
+   - *pos and neg predictive value*: displays the relationship between [positive and negative predictive values](https://en.wikipedia.org/wiki/Positive_and_negative_predictive_values) at changing probability thresholds. Threshold can be set by dragging the vertical line left or right.
+   - *true and false positive rate*: displays the relationship between [TP and FP rate](https://en.wikipedia.org/wiki/False_positives_and_false_negatives) at changing probability thresholds. Threshold can be set by dragging the vertical line left or right.
+4. If *Apply Automatically* is ticked, changes are communicated automatically. Alternatively, click *Apply*.
+
+Examples
+--------
+
+At the moment, the only widget which gives the right type of signal needed by the **Calibration Plot** is [Test and Score](../evaluate/testandscore.md). The Calibration Plot will hence always follow Test and Score.
+
+Here is a typical example on the iris data, where we compare two classifiers (namely [Logistic Regression](../model/logisticregression.md) and [Random Forest](../model/randomforest.md)) and input them into [Test and Score](../evaluate/testandscore.md). Test and Score displays evaluation results for each classifier. Then we draw **Calibration Plot** to further analyze the performance of the classifiers. **Calibration Plot** enables you to see prediction accuracy of class probabilities in a plot.
+
```

Comment:
I would add

Judging by the observed curve, the classifier is overly "cautious". Even when it predicts probabilities of around 0.4, the actual class probability is still 0. Conversely, when the classifier is only 0.6 certain that the class is positive, the actual probability of positive class is already almost 1.